WordPress/WordPressApi/WordPressOrgXMLRPCApi.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 import wpxmlrpc
 
-public class WordPressOrgXMLRPCApi: NSObject
+public class WordPressOrgXMLRPCApi: NSObject, WordPressOrgXMLRPC
 {
     public typealias SuccessResponseBlock = (AnyObject, NSHTTPURLResponse?) -> ()
     public typealias FailureReponseBlock = (error: NSError, httpResponse: NSHTTPURLResponse?) -> ()
```

Comment:
Sure, maybe it's good to put things like that in a style guide because it's difficult for a new comer to know in advance the team preferences.